src/components/calcite-slider/calcite-slider.scss

```diff
@@ -131,6 +126,24 @@ $tick-height: 4px;
 .thumb--minValue {
   transform: translate(-7px, -8px);
 }
+:host([debug]) {
+  .thumb {
```

Comment:
you may not need this level of selector. Depending on the specificity you might be able to get away with just the `:host([debug]) .handle__label` (not that it super matters...)

Reply:
Good call, looks like it does work fine without the `.thumb` nesting level.  Just pushed that up.